tools/deps_info.py

```diff
@@ -141,7 +141,6 @@
   'emscripten_set_visibilitychange_callback_on_thread': ['malloc', 'free'],
   'emscripten_set_wheel_callback_on_thread': ['malloc', 'free'],
   'emscripten_webgl_create_context': ['malloc'],
-  'emscripten_webgl_destroy_context': ['emscripten_webgl_make_context_current', 'emscripten_webgl_get_current_context'],
```

Comment:
unrelated?

Reply:
This change is needed for this PR, but it could be split out and landed first if you like.

The reason is this:

Neither `emscripten_webgl_make_context_current` nor  `emscripten_webgl_get_current_context` are defined in native code.  They are both defined in JS:

https://github.com/emscripten-core/emscripten/blob/8c34d43d4407463aba9e196a55af2bf392692253/src/library_html5_webgl.js#L223

https://github.com/emscripten-core/emscripten/blob/8c34d43d4407463aba9e196a55af2bf392692253/src/library_html5_webgl.js#L57


What is more the dependency between `emscripten_webgl_destroy_context` and these two functions is expressed already in JS:

https://github.com/emscripten-core/emscripten/blob/8c34d43d4407463aba9e196a55af2bf392692253/src/library_html5_webgl.js#L313

Dependencies between JS functions and other JS functions don't make any sense in deps_info.py so I'm not sure why this existed.

Prior to this PR such dependencies were harmless but went unnoticed..  with this PR these get turned into required native exports which will cause wasm-ld to fail (quite rightly) with "undefined symbol".

Do you want me to split this out even though it is needed for to this change? 


Having this unneeded

Reply:
Split out into https://github.com/emscripten-core/emscripten/pull/15719